app/controllers/test_messages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class TestMessagesController < ApplicationController
+  def new
+    @message = ScheduledMessage.find(params[:scheduled_message_id])
+  end
+
+  def create
+    message = ScheduledMessage.find(params[:scheduled_message_id])
+    employee = Employee.find_by(slack_username: slack_username)
+
+    if employee
+      MessageSender.new(employee, message).delay.run
```

Comment:
I think this should just be `MessageSender.new(employee, message).run`? When I tested it, it only works when I remove delay.